nessie/lakectl_util.go

```diff
@@ -176,10 +177,11 @@ func runCmdAndVerifyResult(t *testing.T, cmd string, expectFail bool, isTerminal
 }
 
 var (
-	timeStampRegexp = regexp.MustCompile(`timestamp: \d+\r?\n`)
-	timeRegexp      = regexp.MustCompile(`\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} [-+]\d{4} \w{1,3}`)
-	commitIDRegExp  = regexp.MustCompile(`[\d|a-f]{64}`)
-	checksumRegExp  = regexp.MustCompile(`[\d|a-f]{32}`)
+	timeStampRegexp     = regexp.MustCompile(`timestamp: \d+\r?\n`)
+	timeRegexp          = regexp.MustCompile(`\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} [-+]\d{4} \w{1,3}`)
+	commitIDRegExp      = regexp.MustCompile(`[\d|a-f]{64}`)
+	shortCommitIDRegExp = regexp.MustCompile(`[\d|a-f]{16}`)
```

Comment:
This might be a dumb question, but can you explain why this is removed (I'm assuming it's connected to `s = normalizeShortCommitID(s)` in line 125), and is this a general fix or directly connected to the annotate command?